okdtest.sh

```diff
@@ -150,7 +150,7 @@ ${KUBERNETES_CLI} get clusterrolebindings --no-headers | awk '/traefik-/{print $
 
 sudo rm -rf ${PV_ROOT}/*
 
-repoLogin
+#repoLogin
```

Comment:
Remove code, do not comment it out.  Git provides the history needed to recover if we ever need to do so.

Reply:
done